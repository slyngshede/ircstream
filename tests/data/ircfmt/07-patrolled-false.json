{
  "$schema": "/mediawiki/recentchange/1.0.0",
  "meta": {
    "uri": "https://ca.wikipedia.org/wiki/George_Blagden",
    "request_id": "52f5f637-7866-43f8-a09b-54f7d8902c22",
    "id": "bcfdcdbf-f59c-4725-b884-36e5b3302d09",
    "dt": "2024-10-01T22:08:17Z",
    "domain": "ca.wikipedia.org",
    "stream": "mediawiki.recentchange",
    "topic": "codfw.mediawiki.recentchange",
    "partition": 0,
    "offset": 1136213545
  },
  "id": 128015173,
  "type": "edit",
  "namespace": 0,
  "title": "George Blagden",
  "title_url": "https://ca.wikipedia.org/wiki/George_Blagden",
  "comment": "",
  "timestamp": 1727820497,
  "user": "Suc de pinya",
  "bot": false,
  "notify_url": "https://ca.wikipedia.org/w/index.php?diff=34067125&oldid=34067114&rcid=128015173",
  "minor": true,
  "patrolled": false,
  "length": {
    "old": 8995,
    "new": 9029
  },
  "revision": {
    "old": 34067114,
    "new": 34067125
  },
  "server_url": "https://ca.wikipedia.org",
  "server_name": "ca.wikipedia.org",
  "server_script_path": "/w",
  "wiki": "cawiki",
  "parsedcomment": ""
}
