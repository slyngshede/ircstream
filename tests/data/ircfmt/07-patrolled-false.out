#ca.wikipedia
14[[07George Blagden14]]4 !M10 02https://ca.wikipedia.org/w/index.php?diff=34067125&oldid=34067114&rcid=128015173 5* 03Suc de pinya 5* (+34) 10
